roles/BCPC-Hadoop-Head.json

```diff
@@ -13,9 +13,9 @@
     "recipe[bcpc::haproxy]",
     "recipe[bcpc::zabbix-head]",
     "recipe[bcpc::diamond]",
```

Comment:
As discussed, we should probably move diamond after graphite; since graphite is needed for diamond to make sense.